tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php

```diff
@@ -210,6 +210,32 @@ public function provideFixCases()
         ->method();
 ?>',
             ],
+            [
+                '<?php
+
+    $user->setFoo(1)
+        ->setBar([
+                1 => 1,
```

Comment:
IMO this line and the one below should be adjusted (removing 4 spaces) as well.

Reply:
This PR isn't related to #3558, there is no new feature here.

It's just a bugfix that, if a call contains multiline contents, `MethodChainingIndentationFixer` should fix it as well.

Reply:
:+1: